src/RevSysCalls.cc

```diff
@@ -620,6 +620,43 @@ EcallStatus RevCore::ECALL_openat() {
   return EcallLoadAndParseString( pathname, action );
 }
 
+// 1024, int rev_open(const char *filename, int flags)
+EcallStatus RevCore::ECALL_open() {
+  auto& EcallState = Harts.at( HartToExecID )->GetEcallState();
+  if( EcallState.bytesRead == 0 ) {
+    output->verbose(
+      CALL_INFO, 2, 0, "ECALL: openat called by thread %" PRIu32 " on hart %" PRIu32 "\n", ActiveThreadID, HartToExecID
+    );
+  }
+  auto pathname = RegFile->GetX<uint64_t>( RevReg::a0 );
+
+  // commented out to remove warnings
```

Comment:
Not sure if the comments in here are related to this function or an artifact of copy/paste. If they are related to `open` then keep em, otherwise probably best to ditch em.

Reply:
comment removed!